package.yaml

```diff
@@ -52,15 +52,16 @@ when:
       # Introduced in GHC 8.
       - -Wredundant-constraints
       # Older Cabal generates warnings with its Paths_* module.
-      - -Wmissing-import-lists
```

Comment:
ignore the changes in this file -- i accidentally brought in the dependencies, and this flag was removed because `freer-effects` has insane coding styles